src/client/js/firebase.js

```diff
@@ -16,12 +16,15 @@
  *  limitations under the License
  *
  */
-var config = {
-  apiKey: "AIzaSyAhJEsYl-waHmh79ReYD1n3Ez6kicjHNT4",
-  authDomain: "devnooktests.firebaseapp.com",
-  databaseURL: "https://devnooktests.firebaseio.com",
-  projectId: "devnooktests",
-  storageBucket: "devnooktests.appspot.com",
-  messagingSenderId: "465166485267"
+const config = {
```

Comment:
Woah. I though I knew better by now than to use var;) Copy-pasting is dangerous. Thanks for correcting.

Reply:
No worries! I blame the fact that I haven't made linting mandatory yet. Too easy to miss stuff without it ;)